ddtrace/tracer/spancontext.go

```diff
@@ -78,6 +80,9 @@ func (c *spanContext) SpanID() uint64 { return c.spanID }
 // TraceID implements ddtrace.SpanContext.
 func (c *spanContext) TraceID() uint64 { return c.traceID }
 
+// TraceID implements ddtrace.SpanContext128TraceID.
+func (c *spanContext) TraceID128() string { return fmt.Sprintf("%d%d", c.traceIDHigh, c.traceID) }
```

Comment:
I think these need to be hex-formatted, otherwise concatenation will not yield the correct integer. 
Also, I think hexadecimal is how they are formatted in the header, so that would keep it more consistent.